CMakeLists.txt

```diff
@@ -853,6 +853,10 @@ endif()
 
 if(CONFIG_CODE_DATA_RELOCATION)
    set(MEM_RELOCATAION_LD "${PROJECT_BINARY_DIR}/include/generated/linker_relocate.ld")
```

Comment:
Isn't here a typo in the cmake variable?

Reply:
> Isn't here a typo in the cmake variable?
yes, it should be RELOCATION. I will change it with next patch.